schema/swagger.yml

```diff
@@ -179,32 +179,36 @@ definitions:
   NodeStatus:
     type: object
     required:
-      - syncing
+      - broadhash
+      - consensus
       - height
+      - loaded
       - networkHeight
-      - consensus
-      - broadhash
+      - syncing
```

Comment:
Any specific reason to change the order of attributes under `required`. As per swagger it don't have any impact, but it makes difficult to identify in PR that which attribute was newly added.

Reply:
Introduce a non-existent order is a good reason to me.

Reply:
Can't understand `non-existent` ? The order in required attributes is nothing to do with implementation or params validation.